### PR TITLE
Image gallery cleanup

### PR DESCRIPTION
**Pull request by dho1994**
Finally a working expanded view for the main image. Thumbnail carousel that displays big images and little dots depending on the view. Dynamically updating cursor image with picture toggling functionality. Lots of styling updates, new colorways and a new banner option you can play around with.

### CODE REVIEW

File: client/src/components/ProductOverview/OverlayCarousel.jsx [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
+          {showUpArrow && (
+            <div id="carousel-up-one" className={upDownArrowClass}>
+              <i className="fas fa-angle-up" onClick={handleScrollUp} role="presentation" />
+            </div>
+          )}
+
+          {currCarouselView.map((photo) => (
+            <OverlayThumbnail
+              key={photo.url}
```
Comment (bishalkg) [inline client/src/components/ProductOverview/OverlayCarousel.jsx:76]: noice trick for unique val

File: client/src/components/ProductOverview/OverlayCarousel.jsx [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+          )}
+
+        </div>
+
+      </>
+    </>
+  );
+}
+
+OverlayCarousel.propTypes = {
```
Comment (bishalkg) [inline client/src/components/ProductOverview/OverlayCarousel.jsx:97]: This is really nice, I need to be more exact about my propType Validation

File: client/src/components/ProductOverview/OverlayThumbnail.jsx [hunk 3]
```diff
@@ -12,9 +17,32 @@ function OverlayThumbnail(props) {
 
   let overlayThumbnailClass = 'overlay-thumbnail-image';
   overlayThumbnailClass += isOverlayThumbnailSelected ? '-select' : '';
```
Comment (bishalkg) [inline client/src/components/ProductOverview/OverlayThumbnail.jsx:19]: god of manipulating css styles

**Review by bishalkg — APPROVED** [review]
Super clean code and logic, I learned a lot. I like how the Carousel  and Overlaythumbnail are stateless.

**Review by ya-liu — APPROVED** [review]

**Pull request merged**